db/migrate/20190830134723_add_deleted_at_to_spree_prices.rb

```diff
@@ -0,0 +1,6 @@
+class AddDeletedAtToSpreePrices < ActiveRecord::Migration
+  def change
+    add_column :spree_prices, :deleted_at, :datetime
+    add_index :spree_prices, :deleted_at
```

Comment:
I think we've discussed this before but indexes on boolean columns are little to no value and make writes slower. I wouldn't add it.

Reply:
It was recommended in the paranoia gem's documentation, but I can remove it :+1:

Reply:
Removed :+1:

Reply:
> I think we've discussed this before but indexes on boolean columns add little to no value and make writes slower. I wouldn't add it.

I didn't know. This is useful information, thanks. @sauloperez